end2end/tests/Workflow Editor - Creating and Editing events.spec.ts

```diff
@@ -0,0 +1,131 @@
+import{test, expect} from "../fixtures/MyCustomWorkflowFixtures"
+
+test.use({
+  ignoreHTTPSErrors: true
+});
+
+
+test.beforeAll(async ({workflowData})=>{
+  let randNum = Math.random();
+  workflowData.uniqueText = `New Event ${randNum}`;
+  workflowData.uniqueDescr = `Adding description ${randNum}`;
+
+})
+
+//Create a New Event
+test ('CreateEvent', async ({ page, workflowData}, testinfo) => {
+ await page.goto('https://host.docker.internal/Test_Request_Portal/admin/?a=workflow');

```

Comment:
This line, and similar lines in other tests are redundant.

Reply:
remove the first line from all test